README.md

```diff
@@ -1,24 +1,14 @@
 # README
 
-This README would normally document whatever steps are necessary to get the
-application up and running.
+Thanks for taking a look at this!
 
-Things you may want to cover:
+A couple of questions:
 
-* Ruby version
+* Is there any good way to add js to ransack?  Trying to get it to find without reloading.  I've heard re-rendering
+the contact loop on click could work.  What would that look like?
 
-* System dependencies
+* In terms of adding contacts/groups I had to be clever to get them to adjust all the others in the grid
+so i layered an index to identify what is "0" and then put the new one before it. This does come with risk though
+as if a user deletes "contact_card_0" then the code won't run.  is there a better way to work this?
```

Comment:
Ah, I see. You're referring to this line?

https://github.com/jrohkohl/contactify/blob/90b0eaabcfba63fc09dc653308997c420066aebd/app/views/contacts/create.js.erb#L5

Since it's possible there isn't even an element with the id `contact_card_0`, I would instead add a class to the containing div, something like

```erb
<%# in contacts/index.html.erb %>
  <div class="row mt-3 contact-list">
    <% @contacts.by_created_date.each_with_index do |contact, index| %>
      <%= render "contacts/contact", contact: contact, index: index %>
    <% end %>
  </div>
```

and then update the `create.js.erb` to prepend to that class

```erb
var added_contact = $("<%= j(render "contacts/contact", contact: @contact, index: nil) %>");

added_contact.hide();

$(".contact-list").prepend(added_contact);
```